A3A/addons/core/functions/CREATE/fn_createVehicleCrew.sqf

```diff
@@ -69,6 +69,10 @@ private _fnc_addCrewToTurrets = {
 		if (!_isHeli && {getNumber (_turretConfig >> "showAsCargo") > 0}) then { continue };
 		if (isNull (_vehicle turretUnit _turretPath)) then {
 			private _gunner = [_group, _unitType, getPos _vehicle, [], 10] call A3A_fnc_createUnit;
+			/* if (_gunner == objNull) then {
+				_gunner = [_group, _unitType, getPos _vehicle, [], 10] call A3A_fnc_createUnit;
+			};
+			FactionGetTiered(inv,"unitRifle"), */
```

Comment:
Why is this even needed?

Reply:
I was experimenting, trying to find what was causing the problem